cmd/agent/gui/views/private/js/codemirror.js

```diff
@@ -1645,7 +1645,7 @@ function extractLineClasses(type, output) {
     var prop = lineClass[1] ? "bgClass" : "textClass"
     if (output[prop] == null)
       { output[prop] = lineClass[2] }
-    else if (!(new RegExp("(?:^|\s)" + lineClass[2] + "(?:$|\s)")).test(output[prop]))
```

Comment:
I suspect this was supposed to be `\\s`, but I'm not sure :)

Reply:
https://github.com/stephengroat/datadog-agent/pull/1/commits/fb14adbc126746442c73b968d30dfb3caa3694ae

honestly, i had/have no clue what this regex is searching for, i was just blindly following the advice of the tool.

Reply:
At the risk of ruining a joke, that's one of the risks of static-analysis tools like this!